pkg/support/dump_test.go

```diff
@@ -38,3 +46,38 @@ func TestParseFileName(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, ts.String(), "2021-08-17 09:47:58 +0000 UTC")
 }
+
+func TestTimestampFilter(t *testing.T) {
+	since := "7s"
+	ts := timestampFilter(since)
+	currentTime := time.Now()
+	assert.Contains(t, ts.String(), currentTime.Format("2006-01-02"))
```

Comment:
"since" is used only to call the function , and for matching exact output it includes time also so every time it changes due to seconds, minutes ,so I tried matching with the date format

Reply:
how the time since = 7s is calculated for this test !

Reply:
It is not calculated ,`timestampFilter` gives the time when dump have started ,to get that exact time ,the time specified as since is subtracted from current time( `time.Now()` ) so for testing purpose it is given as "7s"(7 seconds) we can also give "1h", "3m" etc...

Reply:
I agree that the test is not great as it is. Something like this would make more sense IMO:
```go
ts := timestampFilter("2m")
currentTime := time.Now()
require.NotNil(t, ts)
expectedTime := currentTime.Add(-2*time.Minute)
assert.WithinDuration(t, expectedTime, *ts, 10*time.Second)
```